bridge_adaptivity/bridge_lti/validator.py

```diff
@@ -100,4 +100,14 @@ def get_client_secret(self, client_key, request):
         present, or None if the key does not exist in the database.
         """
         log.debug('Getting client secret')
-        return self.lti_consumer.consumer_secret
+        return self.lti_consumer.consumer_secret if client_key else None
+
+    @property
+    def dummy_client(self):
+        """
+        Dummy client used when an invalid client key is supplied.
+
+        :return: Always `None`
+        """
+        log.debug('Getting dummy client')
```

Comment:
I would change log text to: `[Warning] Client key validation failed, getting dummy client then`